src/main/java/seedu/duke/command/ListMenuCommand.java

```diff
@@ -1,17 +1,30 @@
 package seedu.duke.command;
 
 import seedu.duke.data.Menu;
-//import seedu.duke.data.dish.Dish;
 import seedu.duke.ui.Ui;
+import seedu.duke.ui.UserOutput;
+
+import java.text.DecimalFormat;
 
 /**
  * Lists all dishes in the menu to the user.
  */
 public class ListMenuCommand extends Command {
+    private static final DecimalFormat dollarValue = new DecimalFormat("0.00");
+    /**
+     * Iterates through the menu arraylist, outputting the dish name and dish price.
+     *
+     * @param menu ArrayList of Dishes
+     * @param ui Handles the interactions with user
+     */
     @Override
     public void execute(Menu menu, Ui ui) {
-        /*for (String d : menu) {
-            System.out.println(d);
-        }*/
+        ui.showToUser(UserOutput.LIST_MESSAGE.message);
+        for(int i =0; i < menu.getSize(); i++) {
```

Comment:
take note of the spacing
```suggestion
        for(int i = 0; i < menu.getSize(); i++) {
```